packages/core/src/config/__tests__/config-resolvers.test.ts

```diff
@@ -259,11 +261,13 @@ describe('resolveConfig', () => {
     const rootOrApiRawConfig = {
       extends: ['local-config-with-custom-function.yaml'],
     };
-    const { plugins } = await resolveConfig({
+    const { plugins, resolvedConfig } = await resolveConfig({
       rawConfigDocument: makeDocument(rootOrApiRawConfig, configPath),
       configPath,
     });
 
+    new Config(resolvedConfig, { plugins });
```

Comment:
What does this do?

Reply:
Yes, but I don't see any related expects in the test. Or am I'm missing something?